lib/grover/js/processor.cjs

```diff
@@ -16,35 +16,46 @@ const path = require('path');
 
 const _processPage = (async (convertAction, urlOrHtml, options) => {
   let browser, errors = [], tmpDir;
-  try {
-    tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), 'grover-'));
-
-    const launchParams = {
-      args: process.env.GROVER_NO_SANDBOX === 'true' ? ['--no-sandbox', '--disable-setuid-sandbox'] : [],
-      userDataDir: tmpDir
-    };
-
-    // Configure puppeteer debugging options
-    const debug = options.debug; delete options.debug;
-    if (typeof debug === 'object' && !!debug) {
-      if (debug.headless !== undefined) { launchParams.headless = debug.headless; }
-      if (debug.devtools !== undefined) { launchParams.devtools = debug.devtools; }
-    }
 
-    // Configure additional launch arguments
-    const args = options.launchArgs; delete options.launchArgs;
-    if (Array.isArray(args)) {
-      launchParams.args = launchParams.args.concat(args);
-    }
+  // Configure puppeteer debugging options
+  const debug = options.debug; delete options.debug;
```

Comment:
seems like these two have been moved away from the rest of the related code? Not clear why they shouldn't be down at ~L38?

Reply:
It's used again at the very end of the `try { }` block. Moving it inside the else makes it undefined when using remote browser option

Reply:
Of course, my mistake 😄